Src/Microsoft.Dynamic/zlib.net/Deflate.cs

```diff
@@ -2061,8 +2061,8 @@ internal int deflate(ZStream strm, FlushStrategy f)
 								head[i] = 0;
 						}
 					}
-					strm.flush_pending();
-					if (strm.avail_out == 0)
+					strm.FlushPending();
+                    if (strm.avail_out == 0)
```

Comment:
I think this is because some of these files have mixed tabs and spaces. My changes always use spaces.